RadarSDKTests/CLLocation+RadarTests.swift

```diff
@@ -0,0 +1,64 @@
+//
```

Comment:
Having a mixed source codebase increases complexity for little benefit.  We will rewrite the SDK in Swift probably some time this year, but let's keep things simple in this repo by sticking to Obj-C.

Reply:
Done.